Makefile

```diff
@@ -1,18 +1,30 @@
 
+SHELL:=/usr/bin/env bash
 # Image URL to use all building/pushing image targets
 IMG ?= controller:latest
 # Produce CRDs that work back to Kubernetes 1.11 (no version conversion)
-CRD_OPTIONS ?= "crd:trivialVersions=true,crdVersions=v1beta1"
+CRD_OPTIONS ?= "crd:trivialVersions=true,crdVersions=v1"
 
+TOOLS_DIR := hack/tools
+TOOLS_BIN_DIR := $(TOOLS_DIR)/bin
 BIN_DIR := $(PWD)/tools/bin
 KUSTOMIZE := $(BIN_DIR)/kustomize
 CONTROLLER_GEN := $(BIN_DIR)/controller-gen
 COVER_PROFILE = cover.out
+KUBEBUILDER := $(TOOLS_BIN_DIR)/kubebuilder
+KUSTOMIZE := $(TOOLS_BIN_DIR)/kustomize
 
 all: manager
 
 # Run tests
-test: generate fmt vet manifests unit
+#test: generate fmt vet manifests unit
```

Comment:
Maybe remove this and comment line above, since we have a new test target defined below?

Reply:
Yes sure, removed it.